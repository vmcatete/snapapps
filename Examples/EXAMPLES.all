Day_1_Draw_Square_Begin.xml	Draw Square Starter
Day_1_Draw_Square_Final.xml	Draw Square Final
Day_1_Introduce_Yourself_Begin.xml	Introduce Yourself Starter
Day_1_Introduce_Yourself_Final.xml	Introduce Yourself Final
Day_2_Balancing_Scales_Begin.xml	Balancing Scales Starter
Day_2_Balancing_Scales_Final.xml	Balancing Scales Final
Day_2_Graphing_Coordinates_Begin.xml	Graphing Coordinates Starter
Day_2_Graphing_Coordinates_Final.xml	Graphing Coordinates Final
Day_3_Epidemics_Begin.xml	Epidemics Starter
Day_3_Epidemics_Final.xml	Epidemics Final
Day_3_Foodwebs_Begin.xml	Foodwebs Starter
Day_3_Foodwebs_Final.xml	Foodwebs Final
