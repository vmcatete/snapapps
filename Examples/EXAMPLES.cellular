Day_3_Epidemics_Begin.xml	Epidemics Starter
Day_3_Epidemics_Final.xml	Epidemics Final
Day_3_Foodwebs_Begin.xml	Foodwebs Starter
Day_3_Foodwebs_Final.xml	Foodwebs Final
