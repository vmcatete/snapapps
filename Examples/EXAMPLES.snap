Day_1_Draw_Square_Begin.xml	Draw Square Starter
Day_1_Draw_Square_Final.xml	Draw Square Final
Day_1_Introduce_Yourself_Begin.xml	Introduce Yourself Starter
Day_1_Introduce_Yourself_Final.xml	Introduce Yourself Final
Day_2_Balancing_Scales_Begin.xml	Balancing Scales Starter
Day_2_Balancing_Scales_Final.xml	Balancing Scales Final
Day_2_Graphing_Coordinates_Begin.xml	Graphing Coordinates Starter
Day_2_Graphing_Coordinates_Final.xml	Graphing Coordinates Final

